extensions/vat-calculator/src/noviewnet.jsx

```diff
@@ -0,0 +1,20 @@
+import { Clipboard, showHUD, getPreferenceValues } from "@raycast/api";
+import { runAppleScript } from "@raycast/utils";
+
+import { getNetPrice } from "./lib/vat";
+
+export default async function quickVat(props) {
+  var number = getNetPrice(props.arguments.number);
```

Comment:
I would maybe add [CloseMainWindow](https://developers.raycast.com/api-reference/window-and-search-bar#closemainwindow) to ensure Raycast closes as soon as possible 🙂